<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <arg name="spawn_point" default="[-6.0, -40.0, 2.0, 90.0]" />
    <arg name="vehicle_filter" default="vehicle.toyota.*" />

    <rosparam param="spawn_point" subst_value="True">$(arg spawn_point)</rosparam>
    <param name="vehicle_filter" value="$(arg vehicle_filter)" />

    <node pkg="msl_ego_vehicle" type="msl_ego_vehicle.py" name="msl_ego_vehicle" output="screen"/>
</launch>
